core/src/main/java/org/commonjava/indy/data/DefaultStoreValidator.java

```diff
@@ -42,11 +42,13 @@
     @Inject
     SslValidationConfig configuration;
 
+    final HashMap<String, String> errors = new HashMap<>();
```

Comment:
I think this validator should be operating as @ApplicationScoped, right? If so, then putting these errors as an object-level field is a bad idea, since the errors will bleed across invocations.

Reply:
I will change this .